crates/core/c8y_api/src/smartrest/message.rs

```diff
@@ -174,7 +172,8 @@ mod tests {
 
     #[test]
     fn control_chars_are_removed() {
-        let input = generate_test_vec_u8();
+        let input: Vec<u8> = (0x00..0xff).collect();
+        dbg!(&input);
```

Comment:
remove dbg!

Reply:
Oh my bad, was testing if it will really be the same and forgot to remove it.